src/main/java/model/Race.java

```diff
@@ -0,0 +1,34 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Race {
+
+    private ArrayList<Car> cars = new ArrayList<>();
```

Comment:
이번 기회에 일급 컬렉션에 대해서도 학습해보시면 좋을 것 같아요.

- https://jojoldu.tistory.com/412

그리고 타입은 구체적인 클래스를 사용하기 보단 인터페이스 또는 추상 클래스를 사용하는게 좋답니다. (변경의 용이함 때문)
```suggestion
private List<Car> cars;
```

Reply:
필드로 구현 클래스를 사용하지 않도록 하였습니다